lightning/src/ln/msgs.rs

```diff
@@ -1636,6 +1666,7 @@ mod tests {
 		}
 		target_value.append(&mut hex::decode("009000000000000f42400000271000000014").unwrap());
 		if htlc_maximum_msat {
+			target_value.append(&mut hex::decode("0000000000000001").unwrap());
```

Comment:
This is confusing - the next line now refers exclusively to excess data which is *not* the htlc_maximum_msat. Comment would be fine but separate out the excess data testing from htlc_maximum testing.

Reply:
Sure, I just overlooked this intersection.

Reply:
I fixed the code. Not sure why the comment is useful, I think it's self-explanatory :)